src/App.tsx

```diff
@@ -1,25 +1,51 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+
+  const [perPage, setPerPage] = useState(5)
+  const [page, setPage] = useState<number>(1);
+
+  const total = 42;
+
+  const handlePerPageValueChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setPerPage(Number(event.target.value));
+  }
```

Comment:
The 'handlePerPageValueChange' function directly sets the state with the event target value. It would be safer to parse the value to an integer and check for NaN before setting the state to avoid potential bugs.